bfabric/bfabric_config.py

```diff
@@ -27,73 +30,146 @@ class BfabricConfig:
     """Holds the configuration for the B-Fabric client for connecting to particular instance of B-Fabric.
 
     Attributes:
-        base_url (optional): The API base url
+        webbase (optional): The API base url
         application_ids (optional): Map of application names to ids.
         job_notification_emails (optional): Space-separated list of email addresses to notify when a job finishes.
     """
 
-    base_url: str = "https://fgcz-bfabric.uzh.ch/bfabric"
+    webbase: str = "https://fgcz-bfabric.uzh.ch/bfabric"
     application_ids: Dict[str, int] = dataclasses.field(default_factory=dict)
     job_notification_emails: str = ""
 
     def with_overrides(
         self,
-        base_url: Optional[str] = None,
+        webbase: Optional[str] = None,
         application_ids: Optional[Dict[str, int]] = None,
     ) -> BfabricConfig:
         """Returns a copy of the configuration with new values applied, if they are not None."""
         return BfabricConfig(
-            base_url=base_url if base_url is not None else self.base_url,
+            webbase=webbase if webbase is not None else self.webbase,
             application_ids=application_ids
             if application_ids is not None
             else self.application_ids,
         )
 
 
-def parse_bfabricrc_py(file: io.FileIO) -> Tuple[BfabricConfig, Optional[BfabricAuth]]:
+'''
```

Comment:
Would be great to move this explanation to a sensible location

- Either module level docstring (i.e. just put it at the beginning of the file)
- Or the docstring of the function that does the reading

Reply:
I will put it into the docstring of read_config()